javaScriptAJAX-programming/codeChallenge/roomWidget.js

```diff
@@ -0,0 +1,26 @@
+let getRooms =  new XMLHttpRequest();
```

Comment:
Used `const` for `getRooms` since it doesn't need to be reassigned.

Reply:
293feef36959c25583693e71cc772021fc823ef8